Adding anaerobic digester cost (#1433)
watertap-org/watertap: docs/technical_reference/costing/anaerobic_digester.rst

.. csv-table::
   :header: "Description", "Symbol", "Parameter Name", "Default Value", "Units"

   "Capital cost A parameter :math:`^1`", ":math:`A`", "``capital_a_parameter``", "19.3552312e6", ":math:`\text{USD}_{2012}`"
   "Capital cost B parameter :math:`^1`", ":math:`B`", "``capital_b_parameter``", "0.6", ":math:`\text{dimensionless}`"

>    "Reference flow :math:`^1`", ":math:`F_r`", "``911.054``", "0.6", ":math:`m^3/h`"

Reviewer: ```suggestion
   "Reference flow :math:`^1`", ":math:`F_r`", "``reference_flow``", "911.054", ":math:`m^3/h`"
```

Costing Method Variables
++++++++++++++++++++++++

> There are no costing method variables unique to the CSTR with injection.

Reviewer: ```suggestion
There are no costing method variables unique to the anaerobic digester.
```

Capital Cost Calculations
+++++++++++++++++++++++++

Capital cost is dependent upon the unit's volumetric flowrate, :math:`F`, as shown in the equation below.

    .. math::

>         C_{cap,tot} = A * F/F_r*{B}

Reviewer: ```suggestion
        C_{cap,tot} = A * F/F_r^{B}
```
I think B should be an exponent here

>         C_{cap,tot} = A * F/F_r^{B}

Reviewer 2: ```suggestion
        C_{cap,tot} = A * (F/F_r)^{B}
```

Reviewer 2: I think add the parentheses is helpful to avoid misunderstanding

 
Operating Cost Calculations
+++++++++++++++++++++++++++

Electricity :math:`C_{elec}` is a variable operating cost based on the energy intensity :math:`E` of the unit process

> (electricity consumption for the CSTR with injection), electricity price :math:`P`, electricity flow :math:`Q`, and the plant

Reviewer: ```suggestion
(electricity consumption for the anaerobic digester), electricity price :math:`P`, electricity flow :math:`Q`, and the plant
```
